fdbclient/include/fdbclient/Tenant.h

```diff
@@ -76,11 +93,10 @@ struct TenantMapEntry {
 	static std::string tenantLockStateToString(TenantLockState tenantState);
 	static TenantLockState stringToTenantLockState(std::string stateStr);
 
-	int64_t id = -1;
+	TenantMinimalMetadata tenantMinimalMetadata;
```

Comment:
One option to hide this extra layer in the object is to provide functions on `TenantMapEntry` to access the three variables, such as:

```
int64_t& TenantMapEntry::id() { return tenantMinimalMetadata.id; }
```

Reply:
I wonder why I didn't think of this... Yes that would be much nicer. So sadly I need to refactor everything again :D